js/main.js

```diff
@@ -0,0 +1,15 @@
+const mainMenu = document.querySelector('.mainMenu');
+const openMenu = document.querySelector('.openMenu');
+const closeMenu = document.querySelector('.closeMenu');
+
+function openMobileMenu() {
+  mainMenu.style.display = 'flex';
+  mainMenu.style.top = '0';
+}
+
+function closeMobileMenu() {
+  mainMenu.style.top = '-100%';
+}
```

Comment:
- [ ] Kindly close the menu whenever any nav link is clicked, because now for `portfolio`, `about`, and `contact` it goes to the correct position but the menu is not closed.